src/NHibernate/Dialect/Dialect.cs

```diff
@@ -1410,6 +1410,12 @@ public IList<SqlString> GetTokens()
 		/// </summary>
 		public virtual bool SupportsConcurrentWritingConnectionsInSameTransaction => SupportsConcurrentWritingConnections;
 
+		/// <summary>
+		/// Does this dialect requires null enlistment for reusing a connection after a transaction scope, without
+		/// using a new scope?
+		/// </summary>
+		public virtual bool RequiresNullEnlistment => false;
```

Comment:
Taking it the other way round. That may allow testing only on Dialect instead of testing on driver.